activities/vulnerability.json

```
@@ -0,0 +1,21 @@
+{
+  "tasks": [
+    {
+      "layers": [
+        ["d.rast", "map=water_depth"],
+        ["d.rast", "map=vulnerable", "color=77:77:77", "width=2"]
```

d.rast doesn't have color or width options:
```suggestion
        ["d.rast", "map=vulnerable"]
```